plugins/modules/vmware_host_disk_info.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, Matt Proud 
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: vmware_host_disk_info
+short_description: Gathers information about disks attached to given ESXi host/s.
+description:
+- This module returns information about disks attached to given ESXi host/s
+- If I(cluster_name) is provided, then disk information about all hosts from the given cluster will be returned.
+- If I(esxi_hostname) is provided, then disk information about the given host system will be returned.
+authot:
+- Matt Proud
```

Comment:
```suggestion
author:
- Matt Proud (@laidbackware)
```

Sanity error occurs if there isn’t your GitHub Account at the author.